content/pages/Events.md

```diff
@@ -24,6 +24,5 @@ Members of the CSA gather every Friday night in our Discord, however there’s a
 
 Here is a list of events that the CSA has run, or participated in in the past:
 
-- [Child’s Play Charity Fundraiser](http://web.archive.org/web/20180812130528/http://ufvcsa.ca/childs-play-charity-fundraiser/)
```

Comment:
For posterity's sake, I wasn't able to recover this page from webarchive. It might be present in earlier caches of the website.